src/Resources/config/core.php

```diff
@@ -151,6 +152,23 @@
 
         ->alias(SearchHandler::class, 'sonata.admin.search.handler')
 
+        ->set('sonata.admin.controller.crud', CRUDController::class)
+            ->public()
+            ->tag('container.service_subscriber')
+            ->args([
+                new ReferenceConfigurator('sonata.admin.pool'),
+                new ReferenceConfigurator('sonata.admin.audit.manager'),
+                new ReferenceConfigurator('sonata.admin.object.manipulator.acl.admin'),
+                new ReferenceConfigurator('sonata.admin.breadcrumbs_builder'),
+                new ReferenceConfigurator('request_stack'),
+                new ReferenceConfigurator('translator'),
+                (new ReferenceConfigurator('sonata.exporter.exporter'))->nullOnInvalid(),
+                (new ReferenceConfigurator('sonata.admin.admin_exporter'))->nullOnInvalid(),
+                (new ReferenceConfigurator('sonata.admin.security.acl_user_manager'))->nullOnInvalid(),
+                (new ReferenceConfigurator('logger'))->nullOnInvalid(),
+            ])
+            ->call('setContainer', [new ReferenceConfigurator('service_container')])
```

Comment:
This is not really a good solution. For some reason, the only way to get the container configured to only show the subscriber services is to mark this class as autowired.

Reply:
We could inject the missing services (twig and csrftokenManager, but we still have some internal methods from AbstractController that requires having the container configured

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/6477

also with #6467 and after this PR we can move `CRUDController::*Action` to seperate files.  

@jordisala1991 WDYT? Should I finish #6477 PR?